client/package.json

```diff
@@ -12,9 +12,11 @@
     "port-forward:hub": "kubectl port-forward svc/tackle-hub -n konveyor-tackle 9002:8080",
     "port-forward:keycloak": "kubectl port-forward svc/tackle-keycloak-sso -n konveyor-tackle 9001:8080",
     "port-forward": "concurrently -n keycloak,hub \"npm:port-forward:keycloak\" \"npm:port-forward:hub\"",
+    "port-forward-local": "concurrently -n keycloak \"npm:port-forward:keycloak\"",
     "start:dev:server": "NODE_ENV=development node ../server/index.js",
     "start:dev:client": "GENERATE_SOURCEMAP=true webpack serve --config config/webpack.dev.ts",
     "start:dev": "concurrently -n port-forward,server,client -c \"white.bold.inverse,green.bold.inverse,blue.bold.inverse\" \"npm:port-forward\" \"npm:start:dev:server\" \"npm:start:dev:client\"",
+    "start:local": "concurrently -n port-forward-local,server,client -c \"white.bold.inverse,green.bold.inverse,blue.bold.inverse\" \"npm:port-forward-local\" \"npm:start:dev:server\" \"npm:start:dev:client\"",
```

Comment:
We only need to add that line:
`"start:dev:local": "concurrently -n port-forward,server,client -c "white.bold.inverse,green.bold.inverse,blue.bold.inverse\" \"npm:port-forward:keycloak\" "npm:start:dev:server\" \"npm:start:dev:client\"",`